### ISSUE TITLE
XPath: Comment in XML file causes error

### ISSUE THREAD

**Issue by duesentrieb17** [dnGrep/dnGrep#451]
The comment in the XML file in
[example.zip](https://github.com/dnGrep/dnGrep/files/5329044/example.zip)
with any Xpath expression causes the following error
![dnGREP](https://user-images.githubusercontent.com/34717473/95112930-e61f1f00-0741-11eb-9e4e-7f366969d940.png)

The error log:
```xml
<Error time="10.5.2020 19:24:02.645">
  <Message>Der Objektverweis wurde nicht auf eine Objektinstanz festgelegt.</Message>
  <Exception>System.NullReferenceException
   bei dnGREP.WPF.FormattedGrepResult.&lt;&gt;c.&lt;.ctor&gt;b__48_0(GrepMatch r)
   bei System.Linq.Enumerable.WhereSelectListIterator`2.MoveNext()
   bei System.Linq.Enumerable.&lt;DistinctIterator&gt;d__64`1.MoveNext()
   bei System.Linq.Enumerable.Count[TSource](IEnumerable`1 source)
   bei dnGREP.WPF.FormattedGrepResult..ctor(GrepSearchResult result, String folderPath)
   bei dnGREP.WPF.ObservableGrepSearchResults.AddRange(List`1 list)
   bei dnGREP.WPF.MainViewModel.SearchProgressChanged(Object sender, ProgressChangedEventArgs e)</Exception>
</Error>
```

**Comment by doug24** [comment]
Yes, definitely a bug, looks like it's been around a while, but I think it won't be difficult to fix.

**Comment by doug24** [comment]
Fixed in 2.9.277
